### ISSUE TITLE
AZR-000442 - Azure.AppService.AvailabilityZone fails with correct config.

### ISSUE THREAD

**Issue by tsc-buddy** [microsoft/PSRule#1900]
**Description of the issue**

AZR-000442 - Azure.AppService.AvailabilityZone is failing tests regardless of when a SKU that supports AZs is used (P1v3) and the ZoneRedundant Property is 'true'. Historically this test was passing

**To Reproduce**

Steps to reproduce the issue:

You can see the pipeline run failures here: https://github.com/tsc-buddy/bicep-registry-modules/actions/runs/10521160585/job/29151334567

**Expected behaviour**

When using SKUs that support AZs (P1v3, P2v3 for example) and zoneRedundant: true, the test should pass.

Note, the SKU Capacity is also set to 3.
**Error output**

``` ***wsfmin001 : Microsoft.Web/serverfarms [1/2]

    [PASS] Azure.AppService.PlanInstanceCount (AZR-000071)
    [FAIL] Azure.AppService.AvailabilityZone (AZR-000442)
    | Template: avm/res/web/serverfarm/tests/e2e/defaults/main.test.bicep:452:31

    | RECOMMEND:
    | Consider using enabling zone redundancy using availability zones to improve the
    | resiliency of your solution.

    | REASON:
    | - The app service plan (***wsfmin001) is not deployed with a SKU that supports zone-redundancy.

    | HELP:
    | - https://azure.github.io/PSRule.Rules.Azure/en/rules/Azure.AppService.AvailabilityZone/

Error: AZR-000442: ***wsfmin001 failed Azure.AppService.AvailabilityZone. Deploy app service plan instances using availability zones in supported regions to ensure high availability and resilience.`

https://github.com/tsc-buddy/bicep-registry-modules/actions/runs/10521160585/job/29151334567

```

**Module in use and version:**

- Module: PSRule.Rules.Azure
- Version: 1.39.0-B0029

Captured output from `$PSVersionTable`:

```

```

**Additional context**

**Comment by BernieWhite** [comment]
@tsc-buddy Fix will ship in v1.39.0.

**Comment by BernieWhite** [comment]
@tsc-buddy PSRule for Azure - Pre-release 1.39.0-B0072 includes this fix.
